src/config/redis-info.ts

```diff
@@ -1,14 +1,10 @@
 import Redis from "ioredis";
+import isProd from "../jira/util/isProd";
 
-export default (connectionName: string): RedisInfo => ({
-	redisOptions: {
-		port: Number(process.env.REDIS_BOTTLENECK_PORT) || 6379,
-		host: process.env.REDIS_BOTTLENECK_HOST || "127.0.0.1",
-		db: 0,
-		connectionName
-	}
+export default (connectionName: string): Redis.RedisOptions => ({
+	port: Number(process.env.REDISX_CACHE_PORT) || 6379,
+	host: process.env.REDISX_CACHE_HOST || "127.0.0.1",
+	db: 0,
+	tls: isProd() ? { checkServerIdentity: () => undefined } : undefined,
```

Comment:
How does it work? If it is prod, then it will be a function, ortherwise undefined?

Reply:
`isProd` should probably be renamed to `isNodeProd`, it's not specific to our deployment environments, but about if we set `NODE_ENV` to `production` which we do we ddev, stg and prod.  In our local, we do `NODE_ENV=development`.  We don't have TLS enabled on our local environments, hence we disable TLS - we only have it enabled for our deployment environments.

The `checkServerIdentity` is a workaround for ioredis to be able to communicate with AWS elasticache for Redis because AWS uses a host which redis uses IPs which would mismatch and prevent connecting to it.

Reply:
Ah got it now. 

>isProd should probably be renamed to isNodeProd
I'd rather suggest to isLocal, so we won't confues two types of prod (no need to do it now)